ludwig/utils/defaults.py

```diff
@@ -130,7 +130,40 @@ def get_default_optimizer_params(optimizer_type):
         raise ValueError('Incorrect optimizer type: ' + optimizer_type)
 
 
+def _perform_sanity_checks(model_definition):
+    assert 'input_features' in model_definition, "Model definition doesn't " \
+                                                 "define any input features."
+
+    assert 'output_features' in model_definition, "Model definition doesn't " \
+                                                  "define any output features."
+
+    assert isinstance(model_definition['input_features'], list), \
+        "Ludwig expects input features in a list. Check your model " \
+        "definition format"
+
+    assert isinstance(model_definition['output_features'], list), \
+        "Ludwig expects output features in a list. Check your model " \
```

Comment:
Check PEP8 guidelines on usage of `\` to break long lines

Reply:
Will update to use ()